src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/tests/RadioGroup.test.js

```diff
@@ -4,7 +4,14 @@ import React from 'react';
 import RadioGroup from '../RadioGroup';
 import Radio from '../Radio';
 
-jest.mock('../Radio');
+jest.mock('../Radio', () => {
+    const {PureComponent} = require.requireActual('react');
```

Comment:
instead of that you could make a plain function, check https://github.com/sulu/sulu/blob/a2fefa99156253cb398c8be033f00d20f5cde822/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/tests/Application.test.js#L6.
```